slices/fonts-dejavu-mono.yaml

```diff
@@ -0,0 +1,19 @@
+package: fonts-dejavu-mono
+
+slices:
+  fonts:
+    contents:
+      /etc/fonts/conf.avail/20-unhint-small-dejavu-lgc-sans-mono.conf:
+      /etc/fonts/conf.avail/20-unhint-small-dejavu-sans-mono.conf:
+      /etc/fonts/conf.avail/57-dejavu-sans-mono.conf:
+      /etc/fonts/conf.avail/58-dejavu-lgc-sans-mono.conf:
+
+      /etc/fonts/conf.d/20-unhint-small-dejavu-lgc-sans-mono.conf:
+      /etc/fonts/conf.d/20-unhint-small-dejavu-sans-mono.conf:
+      /etc/fonts/conf.d/57-dejavu-sans-mono.conf:
+      /etc/fonts/conf.d/58-dejavu-lgc-sans-mono.conf:
+
+      /usr/share/fonts/truetype/dejavu/DejaVuSansMono-Bold.ttf:
+      /usr/share/fonts/truetype/dejavu/DejaVuSansMono-BoldOblique.ttf:
+      /usr/share/fonts/truetype/dejavu/DejaVuSansMono-Oblique.ttf:
+      /usr/share/fonts/truetype/dejavu/DejaVuSansMono.ttf:
```

Comment:
Please include the copyright slice and its reference similary to what has been suggested in fontconfig-config slice.